CONTRIBUTING.md

```diff
@@ -68,6 +68,17 @@ If successful, it will modify `activities.json` with the new specification. Chec
 }
 ```
 
+### Asking Mozilla to Update a Position
+
+Specifications might be updated after a position is decided. Especially if those
+updates are in response to feedback provided by Mozilla. If you think that
+changes to a proposal warrant an updated position from Mozilla, please request
+another review by commenting on the original issue for the specification. Please
+include a summary of the changes that you believe to be relevant.
+
+If you believe that you have new information that might cause Mozilla to
+reconsider a position, provide that information and ask a coordinator to reopen
+the issue.
```

Comment:
The second paragraph reads like it's just an alternative version of the first with slightly different advice.  Maybe merge the "ask a coordinator to reopen the issue" in to the end of the next-to-last sentence of the first paragraph and drop the rest?

("coordinator" also isn't all that well-defined...)